napari_cellseg3d/models/model_TRAILMAP_MS.py

```diff
@@ -0,0 +1,21 @@
+from napari_cellseg3d.models.unet.model import UNet3D
+
+
+def get_weights_file():
+    # original model from Liqun Luo lab, transferred to pytorch
```

Comment:
this note is incorrect; it should say TRAILMAP ported to PyTorch, and trained on mesoSPIM data from EPFL/Wyss